site/content/docs/5.3/migration.md

```diff
@@ -76,6 +76,9 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
 
   - <span class="badge text-warning-emphasis bg-warning">Deprecated</span> The `.list-group-variant()` mixin is now deprecated. We now [use the Sass loop]({{< docsref "/components/list-group#sass-loop" >}}) directly to modify the component's default CSS variables for each variant.
 
+- **Modals**
```

Comment:
I'm going to remove this part since we didn't choose to do it for other components + new Sass vars.

Reply:
Done via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1658/commits/20ee710fa518baf87d9ff5b30ac161aae0b90b95